.changeset/twenty-bears-live.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
+---
+
+Adds a new top-level configuration option to the Astro configuration
```

Comment:
this isn't true any more, we removed that and instead augment in the integration.